.github/workflows/main.yml

```diff
@@ -1,43 +1,36 @@
 name: NetApp Ansible CI
 
-on: [push, pull_request]
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 6 * * *'
 
 jobs:
-  build:
-
+  sanity:
+    name: Sanity (${{ matrix.ansible }} on ${{ matrix.collection }})
     runs-on: ubuntu-latest
     strategy:
-      max-parallel: 6
       matrix:
+        ansible:
+          - stable-2.9
+          - stable-2.10
+          - stable-2.11
+          - devel
         collection: [aws, azure, cloudmanager, elementsw, ontap, um_info]
+
     steps:
-    - uses: actions/checkout@v1
-    - name: Setting up ansible test on ${{ matrix.collection }}
-      uses: actions/setup-python@v1
-      with:
-        python-version: 3.7
-    - name: Install dependencies
-      run: |
-        python -m pip install --upgrade pip
-        pip install ansible==4.0.0a3
-        cd ansible_collections/netapp/${{ matrix.collection }}
-        pip install -r requirements.txt
-    - name: Run Ansible Sanity
-      run: |
-        cd ansible_collections/netapp/${{ matrix.collection }}
-        ansible-test sanity --docker=default
-    - name: Run unit tests
-      run: |
-        cd ansible_collections/netapp/${{ matrix.collection }}
-        ansible-test units --docker=default --python 3.8
-    - name: Ran ansible-test on ansile DEV
-      run: |
-        git clone https://github.com/ansible/ansible/
-        which ansible-test
-        source ansible/hacking/env-setup
-        which ansible-test
-        cd ansible_collections/netapp/${{ matrix.collection }}
-        ansible-test sanity --docker=default
-        ansible-test units --docker=default --python 3.8
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
 
+      - name: Install ansible (${{ matrix.ansible }})
+        run: pip install https://github.com/ansible/ansible/archive/${{ matrix.ansible }}.tar.gz --disable-pip-version-check
 
+      - name: Run sanity tests (${{ matrix.collection }})
+        run: ansible-test sanity --docker -v --color
+        working-directory: ./ansible_collections/netapp/${{ matrix.collection }}
```

Comment:
is there something preventing us to run the unit tests?
I'm OK if we add them next.